### PR TITLE
Odie Client: consolidate logic into odie

### PR DESCRIPTION
**Pull request by arcangelini**
## Proposed Changes

* Move chat logic into Odie
* Rename Odie component into chat
* Refacors the HelpCenterOdie component

## Why are these changes being made?

This is a setup PR for the integration of Zendesk into Odie and Help Center. We are moving to make Odie responsible for the chat state between AI and Human support. This PR changes the name of the Help Center component to be more generic and self explanatory.

## Testing Instructions

Test for regression

### CODE REVIEW

File: packages/help-center/src/components/help-center-chat.tsx [hunk 1]
```diff
@@ -0,0 +1,58 @@
+/* eslint-disable no-restricted-imports */
+/**
+ * External Dependencies
+ */
+import { recordTracksEvent } from '@automattic/calypso-analytics';
+import OdieAssistantProvider, { OdieAssistant } from '@automattic/odie-client';
+import { useEffect } from 'react';
```
Comment (escapemanuele) [inline packages/help-center/src/components/help-center-chat.tsx]: We should use `@wordpress/element` instead

File: packages/odie-client/src/components/message/direct-escalation-link.tsx [hunk 2]
```diff
@@ -1,47 +1,31 @@
 import { __ } from '@wordpress/i18n';
+import { useCallback } from 'react';
```
Comment (escapemanuele) [inline packages/odie-client/src/components/message/direct-escalation-link.tsx]: @wordpress/element

**Review by escapemanuele — APPROVED** [review]
Tested and the code changes look great ✌️

**Comment by a8ci18n** [comment]
This Pull Request is now available for translation here: https://translate.wordpress.com/deliverables/16988456

Some locales (Hebrew, Japanese) have been temporarily machine-translated due to translator availability. All other translations are usually ready within a few days. Untranslated and machine-translated strings will be sent for translation next Monday and are expected to be completed by the following Friday.

Hi @arcangelini, could you please edit the description of this PR and add a screenshot for our translators? Ideally it'd include this string: `Not what you were looking for?`

Thank you in advance!

**Comment by a8ci18n** [comment]
Translation for this Pull Request [has now been finished](https://translate.wordpress.com/deliverables/overview/16988456).

**Pull request merged**